Overview.bs

```diff
@@ -338,42 +338,29 @@ the user agent <em>must</em> run the following steps:
 
 Creating an Element Proxy {#creating-element-proxy}
 ====================================================
-An <a>element proxy</a> can be constructed in the document scope using the {{ElementProxy}}
-constructor. <a>element proxy</a> is a <a>clonable object</a> and thus can be serialized in a
-message and posted to any <a>animator instance</a> via the <a>animator object</a> port.
+<a>Element Proxy</a> objects are created by the user agent for elements which have either 'animator'
+or 'animator-root' CSS properties. They cannot be constructed from user script. A single element may
+have multiple <a>Element Proxy</a> objects associated with it; one per <a>animator instance</a>.
 
-The {{ElementProxy}} constructor takes two parameters, first the element which is being proxied,
-and second a list of {{DOMString}} identifying all of the <a>animatable attribute</a>
-which may be read or mutated using this proxy.
+The {{ElementProxy}} interface exposes two maps: a read only map of proxied input properties to their
+values, and a map of proxied output properties to their values. The input and output properties in
+the maps are those specified in <a>registerAnimator</a> for the given <a>animator instance</a>.
 
 <pre class='idl'>
 [
-    Exposed=(Window,AnimationWorklet),
-    RaisesException=Constructor,
-    Constructor(Animator animator, Element element, sequence<DOMString> proxiedAttributes)
+    Exposed=(AnimationWorklet),
+    NoInterfaceObject,
```

Comment:
Using NoInterfaceObject seems to be advised against in general. Let's allow the interface to be exposed for now and can have an issue about whether to add NoInterfaceObject.